code/modules/mob/language.dm

```diff
@@ -393,6 +393,19 @@
 	flags = RESTRICTED | HIVEMIND
 	follow = 1
 
+
+/datum/language/terrorspider
+	name = "TerrorSpider"
+	desc = "Terror spiders have a limited ability to commune over a psychic hivemind, similar to xenomorphs."
+	speech_verb = "hisses"
+	ask_verb = "hisses"
+	exclaim_verb = "hisses"
+	colour = "alien"
+	key = "a"
```

Comment:
That is intentional. You can't be both a terrorspider and a xeno at once. Xenos are also the closest thing to terrorspiders that currently exist (though again, still BIG differences). 

Having them use the same color/key is good because it shortens the learning curve (less "what's the key for terrorspider chat?").